### DISCUSSION TITLE
Project reboot for Live 11?

### DISCUSSION THREAD

**Discussion by itsbrex** [carrierdown/mutateful#52] [Q&A]
Curious if anyone would be interested in rebooting this project? Tried spinning it up but no luck so far with Live 11. Could take a stab at it but it doesn't seem like there would be much interest? 

Anyone? 😁

**Comment by carrierdown** [comment]
I haven't had the time to give this project the attention it deserves recently, but it's definitely something I will return to soon. It is not dead. I released a beta of version 1.5 earlier this year with Live 11 support. Have you tried this version?

https://github.com/carrierdown/mutateful/releases/tag/v1.5-beta1
